### PR TITLE
Add unit test for EncryptInsertOnUpdateTokenGenerator

### PR DESCRIPTION
**Pull request by iisheng**
For #15820 .

Changes proposed in this pull request:
- Add unit test for EncryptInsertOnUpdateTokenGenerator#isGenerateSQLToken
- Add unit test for EncryptInsertOnUpdateTokenGenerator#generateSQLTokens

### CODE REVIEW

**Review by terrymanu — COMMENTED** [review]
Please fix the CI

**Comment by iisheng** [comment]
> Please fix the CI



>

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/EncryptInsertOnUpdateTokenGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class EncryptInsertOnUpdateTokenGeneratorTest {
```
Comment (terrymanu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/EncryptInsertOnUpdateTokenGeneratorTest.java:55]: Please add final if class does not design for extension

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/EncryptInsertOnUpdateTokenGeneratorTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        when(result.getCipherColumn("t_user", "mobile")).thenReturn("cipher_mobile");
+        EncryptTable encryptTable = mock(EncryptTable.class);
+        when(encryptTable.getLogicColumns()).thenReturn(Collections.emptyList());
+        EncryptAlgorithm<?, ?> encryptAlgorithm = mock(EncryptAlgorithm.class);
+        when(result.findEncryptor("t_user", "mobile")).thenReturn(Optional.of(encryptAlgorithm));
+        when(result.findEncryptor("t_user", "cipher_mobile")).thenReturn(Optional.of(encryptAlgorithm));
+        when(result.getEncryptValues(null, "db_test", "t_user", "mobile",
+                Collections.singletonList(0))).thenReturn(Collections.singletonList("encryptValue"));
+        return result;
+    }
```
Comment (terrymanu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/rewrite/token/EncryptInsertOnUpdateTokenGeneratorTest.java:131]: Please move the method after method with the first call

**Review by terrymanu — CHANGES_REQUESTED** [review]

**Pull request merged**